# HG changeset patch
# User Rik <rik@octave.org>
# Date 1384975877 28800
#      Wed Nov 20 11:31:17 2013 -0800
# Node ID 303157af491fc12d51030e3019296d81b462ef1b
# Parent  3232956a6081d1d1c94f43f539f84bef39b2c8e6
acinclude.m4: Fix undefined value in macro CHECK_BROKEN_STL_ALGO_H when cross-compiling.

* acinclude.m4: Assign a value to octave_cv_broken_stl_algo_h in all possible
cases.

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -140,17 +140,20 @@ v[6] = 207089;
 std::nth_element (v.begin (), v.begin () + 3, v.end ());
 
 return v[3] == 207089 ? 0 : 1;
     ]])],
     octave_cv_broken_stl_algo_h=no,
     octave_cv_broken_stl_algo_h=yes,
     [case "$GXX_VERSION" in
        *4.8.2*)
-         octave_cv_broken_stl_algo_h=yes,
+         octave_cv_broken_stl_algo_h=yes
+       ;;
+       *)
+         octave_cv_broken_stl_algo_h=no
        ;;
      esac
     ])
     AC_LANG_POP(C++)
   ])
   if test "$GXX" = yes; then
     if test $octave_cv_broken_stl_algo_h = yes; then
       case "$GXX_VERSION" in
